Database_Task.txt

```diff
@@ -1 +1,97 @@
+(1)Creating DATABASE
```

Comment:
Don't apply the number for the heading.
Apply the commenting.

E.g -- Created DATABASE employee